rr_computer_vision/include/lane_detection.hpp

```diff
@@ -6,22 +6,71 @@
 #ifndef __LANE_DETECTION_HPP
 #define __LANE_DETECTION_HPP
 
-
 //ROS
 #include <ros/ros.h>
+#include <sensor_msgs/Image.h>
 
 //OPENCV includes
 #include <opencv2/opencv.hpp>
 
+#include <cv_bridge/cv_bridge.h>
+#include <nav_msgs/OccupancyGrid.h>
+
 //LOCAL
 
 class LaneDetection
 {
-  private:
-
   public:
-    LaneDetection();
+    LaneDetection(ros::NodeHandle nh);
     ~LaneDetection();
+    void Multithreshold(const cv::Mat &input_image, const cv::Mat &bounds, cv::Mat &output_image);
```

Comment:
Are these functions ever called outside of this class? If not please make them private functions